handlers.go

```diff
@@ -160,6 +172,16 @@ func sha1Product(product string) string {
 	return product
 }
 
+func osxEsrProduct(product string) string {
+	if product == "firefox-pkg-latest-ssl" {
+		return "firefox-esr-next-pkg-latest-ssl"
+	}
+	if product == "firefox-latest-ssl" {
+		return "firefox-esr-next-latest-ssl"
+	}
```

Comment:
Extremely slight preference for a constant at top of file rather than embedded in the function logic, but I'm only on the slight preference there.